health-api/src/main/java/com/netflix/runtime/health/api/HealthCheckStatus.java

```diff
@@ -26,26 +26,48 @@
     private final boolean isHealthy;
 
     private final List<Health> healthResults;
+    private final List<Health> suppressedHealthResults;
 
     public HealthCheckStatus(boolean isHealthy, List<Health> indicators) {
         this.isHealthy = isHealthy;
         this.healthResults = new ArrayList<>(indicators);
+        this.suppressedHealthResults = new ArrayList<>();
```

Comment:
Is it necessary to track this as a separate list?  Can 'suppressed' be an attribute of Health instead?